packages/lambdas/src/cw-doc-contribution.ts

```diff
@@ -52,11 +52,18 @@ export const handler = Sentry.AWSLambda.wrapHandler(
       }
 
       console.log(`Key: ${key}`);
-      const url = s3Utils.s3.getSignedUrl("getObject", {
-        Bucket: bucketName,
-        Key: key,
-        Expires: SIGNED_URL_DURATION_SECONDS,
-      });
+      const url = await executeWithRetries(
+        () =>
+          s3Utils.s3.getSignedUrlPromise("getObject", {
+            Bucket: bucketName,
+            Key: key,
+            Expires: SIGNED_URL_DURATION_SECONDS,
+          }),
+        {
+          maxAttempts: 3,
+          initialDelay: 500,
+        }
+      );
```

Comment:
I think its confusing that for most other s3 functions the executeWithRetries is in the s3.ts itself but for this one its here. Can we move this one to s3.ts?

Reply:
I was not going to do it, but I checked the usages of the deprecated S3Utils from the lambdas and was able to remove it and point the usages to Core's version without much effort.

There are other usages of `S3Utils.s3.xxx()`, but I'd rather not act on those because it would require building new functions and updating more logic, it would make this PR larger than it already is and testing even harder.